bgpd/bgp_vty.c

```diff
@@ -3265,12 +3423,85 @@ DEFPY (bgp_administrative_reset,
 	return CMD_SUCCESS;
 }
 
+static int bgp_inst_gr_disable_config_vty(struct vty *vty, struct bgp *bgp,
+					  bool on)
+{
+	int ret = BGP_GR_FAILURE;
+
+	/*
+	 * Update the instance and all its peers, if appropriate.
+	 * Then, inform zebra of BGP's GR capabilities, if needed.
+	 */
+	ret = bgp_gr_update_all(bgp, on ? GLOBAL_DISABLE_CMD
+					: NO_GLOBAL_DISABLE_CMD);
+	VTY_BGP_GR_ROUTER_DETECT_AND_SEND_CAPABILITY_TO_ZEBRA(bgp, bgp->peer,
+							      ret);
+	return ret;
+}
+
+static int bgp_global_gr_disable_config_vty(struct vty *vty, bool on)
```

Comment:
This and bgp_inst_gr_disable_config_vty functions are almost the same with ENABLE version, can't they be both bundled together?